nuntium/templatetags/nuntium_tags.py

```diff
@@ -35,3 +38,15 @@ def join_with_commas(obj_list):
 @register.assignment_tag(takes_context=True)
 def assignment_url_with_subdomain(context, view, subdomain=UNSET, *args, **kwargs):
     return subdomainsurls(context, view, subdomain, *args, **kwargs)
+
+
+@register.simple_tag(takes_context=True)
+def custom_stylesheet(context):
+    output = ''
+    if hasattr(context['request'], 'subdomain') and context['request'].subdomain is not None:
+        path = os.path.join(context['request'].subdomain, 'css/custom.css')
```

Comment:
I'd have a slight preference for the paths to be like:

    css/custom/australia-representatives.css
    css/custom/kenyan-politicians.css

... rather than:

    australia-representatives/css/custom.css
    kenyan-politicians/css/custom.css

... especially if you switch to suggesting they go under `instance/static`, which might have other CSS there at some point.